packages/data/addon/services/bard-metadata.js

```diff
@@ -84,6 +91,7 @@ export default Service.extend({
             this._loadMetadataForType('metric', metadata.metrics);
 
             this.set('metadataLoaded', true);
+            this.loadedDataSources.push(options.dataSourceName);
```

Comment:
Maybe use the actual host uri?

Reply:
Eh we'd have to go and look it up from the config. probably not worth it.